indexer/src/transform.test.ts

```diff
@@ -25,10 +25,15 @@ import {
 } from "./deps.ts";
 import { JsonRpcReceipt } from "./types/receipt.ts";
 
-const jsonData = await Deno.readTextFile(
+const jsonTransactionsData = await Deno.readTextFile(
   "indexer/src/test-data/transactionsData.json",
 );
-const transactionsData = JSON.parse(jsonData);
+const transactionsData = JSON.parse(jsonTransactionsData);
+
+const jsonExpectedTransformData = await Deno.readTextFile(
+  "indexer/src/test-data/expectedTransformData.json",
+);
+const expectedTransformData = JSON.parse(jsonExpectedTransformData);
```

Comment:
Maybe can we have one line of doc for each constant so that we know what we have inside?